brainrender_napari/__init__.py

```diff
@@ -1,4 +1,4 @@
 __version__ = "0.0.1"
 
 
-__all__ = "AtlasViewerWidget"
+__all__ = "BrainRenderWidget"
```

Comment:
Tiniest of tiny comments, there's an inconsistency between `BrainRenderWidget` and `BrainrenderWidget`. I'd vote for the second throughout.